unity-client/Assets/Scripts/MainScripts/DCL/Controllers/CharacterController/DCLCharacterController.cs

```diff
@@ -69,10 +69,10 @@ public class DCLCharacterController : MonoBehaviour
 
 
     // Will allow the game objects to be set, and create the DecentralandEntity manually during the Awake 
-    [HideInInspector] public DCL.Models.DecentralandEntity avatarPositionReference;
-    [HideInInspector] public DCL.Models.DecentralandEntity playerReference;
-    [SerializeField] private GameObject avatarPositionGameObject;
-    [SerializeField] private GameObject playerGameObject;
+    [HideInInspector] public DCL.Models.DecentralandEntity avatarReference;
+    [HideInInspector] public DCL.Models.DecentralandEntity firstPersonCameraReference;
```

Comment:
Can we change these so can't be assigned publicly?
```suggestion
    public DCL.Models.DecentralandEntity avatarReference { get; private set; }
    public DCL.Models.DecentralandEntity firstPersonCameraReference  { get; private set; }
```